django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -5539,7 +5573,7 @@ def test_incipit_search(self):
     def test_cantus_id_search(self):
         chant_with_normal_cantus_id = make_fake_chant(cantus_id="012345")
         chant_with_numerals_in_incipit = make_fake_chant(
-            incipit="0 me! 0 my! This is unexpected!"
+            cantus_id="223455", incipit="0 me! 0 my! This is unexpected!"
```

Comment:
Is this Black compliant?

Also, this diff seems to be comparing to a not fully up-to-date `develop` branch... I don't know why this should be the case. Maybe GitHub will recalculate things when this PR stops being a draft?

Reply:
speaking of which...note that black's 2024 stable style guide was released in the last few days, so an upgrade might be in order.

Reply:
> Is this Black compliant?
> 
> Also, this diff seems to be comparing to a not fully up-to-date `develop` branch... I don't know why this should be the case. Maybe GitHub will recalculate things when this PR stops being a draft?

The potential conflict should be resolved now.